Add: Button to lawsuit form fragment - Link to search corporation online

cotom972 opened the pull request.

gulilo commented on projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java:
```diff
@@ -51,7 +54,13 @@ public View onCreateView(
     scrollView = rootView.findViewById(R.id.ScrollView_formFragment);
-
+    searchCompany = rootView.findViewById(R.id.button_formFragment_searchCompany);
+    searchCompany.setOnClickListener(
```
Let's try to stay consistent. You are creating `confirmForm` button then `searchCompany` button, but setting the click listener to `search` first.
Either move the `confirm` listener up or move the `search` listener down.
I like to deal with each button then move to the next. So finding it, setting it's listener and what ever else, then moving to the next and doing the same.

cotom972 replied:
Sounds good :+1:

gulilo commented on projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java:
```diff
@@ -28,6 +30,7 @@
   EditText receivedDate;
   TextInputEditText userId;
   TextView idValidationError;
+  Button searchCompany;
```
`searchCompany` button is only used in the `OnCreateView` function. It doesn't need to be a class field.
Also true to other views, such as `confirmForm`.

cotom972 replied:
:+1:

gulilo commented on projects/noloan/app/src/main/res/layout/lawsuit_fragment_form.xml:
```diff
@@ -271,8 +287,8 @@
                             android:layout_height="wrap_content"
-                            android:gravity="right"
                             android:focusable="false"
+                            android:gravity="right"
```
What is this change?

cotom972 replied:
No change... I guess just the "line ordering"...

cotom972 commented:
Hey! Thanks for the comments..... :+1: 
Changed it, see if there's something missing.....

\+ Added some future add-on for validating the company's ID as well, small add-on so I added it "on the go" to this PR... Have a look
